scripts/Dockerfile-test

```diff
@@ -0,0 +1,12 @@
+FROM node:12.19.0-alpine3.9
+
+WORKDIR "/code"
+VOLUME ["/code"]
```

Comment:
This assumes the host has a compatible architecture (and node_modules) with node:alpine. I guess that's mostly true with Github Actions agents?

Don't know if it's an issue, but it will probably cause issues when run by developers on a mac host. One of the reasons for using docker extensively in the CI was ease of reproduction for developers (@pshomov).

Guess we just need to document how devs can reproduce different CI issues.

Reply:
The other route we could go would be to dynamically expose random db port for these tests. I felt like this was easier, but if we run into problems with it I'm open to taking another stab at this.